agent/versioner/versioner.go

```diff
@@ -90,7 +90,7 @@ func (v *Versioner) binaryVersion(
 	defer cancel()
 
 	if _, err := v.ef.LookPath(binaryName); err != nil {
-		if err.(*exec.Error).Err == exec.ErrNotFound {
+		if errors.Is(err.(*exec.Error).Err, exec.ErrNotFound) {
```

Comment:
Can it be just `errors.Is(err, exec.ErrNotFound)` ?

Reply:
Since exec.Error is a custom Error with its own logic inside, I would keep the original check to avoid unexpected side effects